stations.csv

```diff
@@ -24893,6 +24893,12 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;is_city;country;
 26861;Heinsberg (Rheinl);heinsberg-rheinl;;;51.065036;6.098877;;f;DE;f;Europe/Berlin;t;;;f;;f;8002721;t;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
 26862;Wien;wien;;;48.2082;16.3738;;t;AT;f;Europe/Vienna;t;ATVIE;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;Vienna;Viena;Vienne;Vienna;Wenen;;;;;;;;;;;
 26863;Gent;gent;;;51.05;3.733333;;t;BE;f;Europe/Brussels;t;;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;BEGET;t;f;;;Ghent;Gante;Gand;Gand;;;;;;;;;;;;
+26864;Chamrousse;chamrousse;;;45.1263617;5.8783216;;t;FR;f;Europe/Paris;t;;;f;;f;;f;u0h0tz;t;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
+26865;Les Allues;les-allues;;;45.396892;6.5662699;;t;FR;f;Europe/Paris;t;;;f;;f;;f;u0hdm2;t;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
+26866;Les Orres;les-orres;;;44.4945176;6.5568703;;t;FR;f;Europe/Paris;t;;;f;;f;;f;sputj5;t;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
+26867;Montauban;montauban;;;44.01667;1.35;;t;FR;f;Europe/Paris;t;;;f;;f;;f;spbfqs;t;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
```

Comment:
Could you attach stations 5804 (Montauban Ville Bourbon) and 10499 (Montauban Gare Routière) to this city?